### ISSUE TITLE
To zap or not to zap? That is the choice

### ISSUE THREAD

**Issue by picasso566** [ahuacate/proxmox-ubuntu-fileserver#5]
When running these scripts as is, there is no choice if we want to zap the media for NAS use. 

I am trying out the scripts and when I create a nas ct even on a single disk, it will take a number of hours to init the drive (below). I want to just comment this out, but it will take me a while to figure out the hierarchy of scripts and which variables to make it work locally. Anyway, for a newb to your project, this is a pain point. 

`            sgdisk --zap $dev >/dev/null 2>&1
            dd if=/dev/zero of=$dev bs=1M status=progress
            wipefs --all --force $dev >/dev/null 2>&1
            info "Zapped, destroyed & wiped device: $dev"
`

**Comment by Steveveepee** [comment]
Agreed, this step takes hours (10+ hours on a 9TB disk!) and is not needed for most simple setups
please allow it to be skipped
